internal/fileutils/file.go

```diff
@@ -161,6 +201,26 @@ func CopyTemplate(boxed, to string, assetsBox *rice.Box, templateData interface{
 	tmplFile.Execute(toFile, templateData)
 }
 
+// CopyTemplateFromFile create file from a template file
+func CopyTemplateFromFile(boxed, to string, templateData interface{}) {
+	templateString, err := ioutil.ReadFile(boxed)
+	if err != nil {
+		log.Errorf("Failed to find template file: %v\n", err)
+		os.Exit(1)
+	}
+	copyTemplateFromString(string(templateString), to, templateData)
+}
+
+// CopyTemplateFromAssetsBox create file from a template asset
+func CopyTemplateFromAssetsBox(boxed, to string, assetsBox *rice.Box, templateData interface{}) {
```

Comment:
I think `ExecuteTemplateFromAssetsBox` would be a better name, as it makes explicit that the template is actually executed. `Copy` doesn't give that impression. Same goes for `CopyTemplateFromFile` and `copyTemplateFromString`.